static/usr/lib/systemd/system/vdsm-network.service.in

```diff
@@ -9,6 +9,8 @@ Type=oneshot
 EnvironmentFile=-/etc/sysconfig/vdsm
 ExecStartPre=@BINDIR@/vdsm-tool dump-bonding-options
 ExecStart=@BINDIR@/vdsm-tool restore-nets
+ExecStart=@BINDIR@/vdsm-tool nwfilter
+ExecStart=@BINDIR@/vdsm-tool dummybr
```

Comment:
This means the commands run once during startup - but before they were running each time you start vdsmd service.

Do we have to run this once or for every start?

Reply:
This will effectively be running with each start